src/test/scala/is/hail/variant/GenomeReferenceSuite.scala

```diff
@@ -93,6 +97,17 @@ class GenomeReferenceSuite extends SparkSuite {
     assert(vas.field("i").typ == TInterval(gr))
   }
 
+  @Test def testDefaultReference() {
```

Comment:
I think you may need to reset the default reference at the end of this test because the HailContext gets reused? Some of the other tests are turning up problems with e.g. Locus(my_genome_reference).